app/models/ShortLinkRepository.scala

```diff
@@ -19,43 +20,68 @@ class ShortLinkRepository @Inject() (dbConfigProvider: DatabaseConfigProvider)(i
     def id = column[Long]("id", O.PrimaryKey, O.AutoInc)
     def redirectToUrl = column[String]("redirect_to_url")
     def token = column[String]("token")
+    def expiration = column[Option[LocalDate]]("expiration_date")
+    def expired = column[Boolean]("expired")
 
-    def * = (id, redirectToUrl, token) <> ((ShortLink.apply _).tupled, ShortLink.unapply)
+    def * = (id, redirectToUrl, token, expiration, expired) <> ((ShortLink.apply _).tupled, ShortLink.unapply)
```

Comment:
Most of the changes here came from updating the `ShortLink` object/data model and propagating those changes through to the API.